esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/opt/rose-suite-metoffice.conf

```diff
@@ -1,7 +1,7 @@
 [template variables]
 DRS_CMIP5="BADC"
 DRS_CMIP6="BADC"
-KGO_ROOT_PATH="/data/users/esmval/KGO/"
+KGO_ROOT_PATH="/data/users/esmval/KGO_v2.6.0/"
```

Comment:
Perhaps this comment out of scope, but consider upgrading this to KGO_v2.9.0?

Reply:
This was done in #3386 and has just been merged into this branch :)